core/templates/dev/head/pages/admin/Admin.js

```diff
@@ -258,6 +258,50 @@ oppia.controller('Admin', [
       });
     };
 
+    $scope.reloadAllCollections = function() {
```

Comment:
Would it be OK to not add all this stuff, and just load the existing demo collection instead as needed? I don't think we should introduce a new construct here just for the sake of e2e tests, and we also want to keep the "surface" of each e2e test as small as possible (so that it changes as little as possible server state). Otherwise there will be collisions (and @arun1595 has run into this in the past).